rock.config.mjs

```diff
@@ -17,4 +17,7 @@ export default {
         ios: platformIOS({sourceDir: isHybrid ? './Mobile-Expensify/iOS' : './ios'}),
         android: platformAndroid({sourceDir: isHybrid ? './Mobile-Expensify/Android' : './android'}),
     },
+    fingerprint: {
+        env: ['USE_NGROK', 'NGROK_URL', 'SECURE_NGROK_URL'],
```

Comment:
Can we just include the whole `.env` file? There are lots of variables in there and I don't think we want to maintain a list of variables that can effect this.

Reply:
Busy with Sentry investigations today, I'll update the PR on tomorrow 👍🏻